### ISSUE TITLE
Can not capture an authorized charge

### ISSUE THREAD

**Issue by tamada1** [omise/omise-magento#126]
### Steps to reproduce
1. Set omise setting "Payment Action" to "Authorize Only"
2. Create order
3. Select "Capture Online" and create invoice data

### Expected result
1. The invoice data is created in Magento and the relevant charge data is captured on omise's dashboard

### Actual result
1. The invoice data can not be created, and the following error message is displayed.

```
Transaction has been declined. Please try again later.
```

2. Even on the omise dashboard charge data is not captured and the following response is left in the log.
```
{
  "object": "error",
  "location": "https://www.omise.co/api-errors#used-token",
  "code": "used_token",
  "message": "token was already used"
}
```

As I think, it seems to be trying to create charge data using tokens that have already been used instead of capture the already existing charge data.

**Comment by jacekstan** [comment]
Hi @tamada1,
Sorry for late response,
do you still face that error?

**Comment by jacekstan** [comment]
If you still face that error, please re-open this issue.

**Comment by selojister** [comment]
@tamada1 You can fix this issue yet? Why this re-open for a long time. I've get same problem too. I need help please.

**Comment by jacstn** [comment]
hi, we have fixed it here:
#182 
but we are some improvements that need to be done.

**Comment by jacstn** [comment]
Fixed in #182
